libcontainer/integration/execin_test.go

```diff
@@ -377,6 +378,73 @@ func TestExecInEnvironment(t *testing.T) {
 	}
 }
 
+func genBigEnv(count int) []string {
+	randStr := func(length int) string {
+		const charset = "abcdefghijklmnopqrstuvwxyz0123456789_"
+		b := make([]byte, length)
+		for i := range b {
+			b[i] = charset[rand.Intn(len(charset))]
+		}
+		return string(b)
+	}
+
+	envs := make([]string, count)
+	for i := 0; i < count; i++ {
+		key := strings.ToUpper(randStr(10))
+		value := randStr(20)
+		envs[i] = key + "=" + value
+	}
+
+	return envs
+}
+
+func BenchmarkExecInBigEnv(b *testing.B) {
```

Comment:
Would you remove these benchmark test code later?

Reply:
Why should I? What's wrong in leaving it?

Reply:
https://github.com/opencontainers/runc/blob/cda4c42056dac11727867726aec08c3d88f3d1a0/libcontainer/integration/execin_test.go#L442

Reply:
Ah! My bad. I wanted some way to check the output and did not finish it.

This is now updated.